Gopkg.toml

```diff
@@ -68,10 +68,18 @@
   name = "github.com/json-iterator/go"
   version = "1.1.4"
 
+[[override]]
+  name = "k8s.io/client-go"
+  branch = "master"
```

Comment:
do we need master or can kubernetes-1.14.0 work?

Reply:
I was having an issue similar to [this](https://github.com/kubernetes/kubernetes/issues/76304), but using master solved it. I'll look more into it to see if we can avoid using master.